docs/en/interfaces/third-party/client-libraries.md

```diff
@@ -44,6 +44,8 @@ ClickHouse Inc does **not** maintain the libraries listed below and hasn’t don
  - [nestjs-clickhouse](https://github.com/depyronick/nestjs-clickhouse)
  - [clickhouse-client](https://github.com/depyronick/clickhouse-client)
  - [node-clickhouse-orm](https://github.com/zimv/node-clickhouse-orm)
+ - [clickhouse-ts](https://github.com/bytadaniel/clickhouse-ts)
+ - [clickcache](https://github.com/bytadaniel/clickcache)
```

Comment:
The CI [isn't happy](https://s3.amazonaws.com/clickhouse-test-reports/55498/5e86362abeb7c988479b6a8e59d306b8c5397fcc/style_check.html). Could you add `clickcache` to `utils/check-style/aspell-ignore/en/aspell-dict.txt` so that it isn't tagged as a spelling error, please?